src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -243,17 +243,29 @@ void AddValueDocumentationParts(ISymbol symbol, IDocumentationCommentFormattingS
 
             private void AddExceptions(ISymbol symbol)
             {
-                var exceptionTypes = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true).ExceptionTypes;
+                const int maxTextSize = 70;
+                var documentationComment = symbol.GetDocumentationComment(GetCompilation(), expandIncludes: true, expandInheritdoc: true);
+                var exceptionTypes = documentationComment.ExceptionTypes;
+                var formatter = Services.GetLanguageServices(_semanticModel.Language).GetRequiredService<IDocumentationCommentFormattingService>();
```

Comment:
Consider moving inside the `if (exceptionTypes.Any())`

Reply:
```suggestion
```